src/wasm-lib/kcl/src/unparser.rs

```diff
@@ -108,6 +109,24 @@ impl NonCodeValue {
     }
 }
 
+impl ImportStatement {
+    pub fn recast(&self, options: &FormatOptions, indentation_level: usize) -> String {
+        let indentation = options.get_indentation(indentation_level);
+        let mut string = format!("{}import ", indentation);
+        for (i, item) in self.items.iter().enumerate() {
+            if i > 0 {
+                string.push_str(", ");
+            }
+            string.push_str(&item.name.name);
+            if let Some(alias) = &item.alias {
+                string.push_str(&format!(" as {}", alias.name));
```

Comment:
We could check for the case where the alias is the same as the real name and remove it

Reply:
Done. 871fc2b3a2f4759945269ce24d4f188d1f6a2e27